plugin.yml

```
@@ -0,0 +1,8 @@
+name: SolarCredit
+main: gg.solarmc.solarcredits.SolarCredit
+version: 1.0
```

```suggestion
version: ${project.version}
```
I suggest using this variable and enabling resource filtering so that the version is automatically set at build-time.